core/templates/dev/head/domain/utilities/language-util.service.ts

```diff
@@ -111,7 +111,8 @@ export class LanguageUtilService {
     return this.getAllAudioLanguageCodes();
   }
   getAudioLanguageDescription(audioLanguageCode: string): string {
-    return this.getSupportedAudioLanguages()[audioLanguageCode].description;
+    const language = this.getSupportedAudioLanguages()[audioLanguageCode];
+    return language ? language.description : null;
```

Comment:
Is this generally a good idea to do in angular? I am not sure... @ankita240796 @bansalnitish can you advise?

Reply:
What do you mean by **this** @nithusha21 ? The ternary condition?

Reply:
I guess its fine (the ternary condition)!